sdk/eventhub/azure-eventhub/tests/perfstress_tests/send_event_batch.py

```diff
@@ -0,0 +1,40 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from ._test_base import _SendTest
+
+from azure_devtools.perfstress_tests import get_random_bytes
+
+from azure.eventhub import EventData
+
+
+class SendEventBatchTest(_SendTest):
+    def __init__(self, arguments):
+        super().__init__(arguments)
+        self.data = get_random_bytes(self.args.event_size)
+
+    def run_sync(self):
+        batch = self.producer.create_batch()
+        for i in range(self.args.num_events):
+            try:
+                batch.add(EventData(self.data))
+            except ValueError:
+                # Batch full
+                self.producer.send_batch(batch)
+                batch = self.producer.create_batch()
+                batch.add(EventData(self.data))
+        self.producer.send_batch(batch)
+
+    async def run_async(self):
+        batch = await self.async_producer.create_batch()
```

Comment:
is this something that can go in setup?
the run_** methods should have a single API ideally

Reply:
please keep the run methods as simple as possible so that we can only test the api call performance - i would split create_batch and send_batch into two different perf tests

Reply:
`create_batch` is majorly a local operation, I don't think we need a perf tests for that :P
but to simplify the `run` and `run_async`, I have taken your suggestion and moved event data batch creation into `__init__`.